resources/eventing/charts/controller/values.yaml

```diff
@@ -42,7 +42,7 @@ metrics:
     nameSuffix: "-metrics"
```

Comment:
do we need this nameSuffix and the nameSuffix in line 46?

Reply:
They were pointing to the same, so I removed them. Also removed some over unused constants.